src/server/middleware/security/oidc_cli.go

```diff
@@ -65,6 +65,11 @@ func (o *oidcCli) Generate(req *http.Request) security.Context {
 
 	info, err := oidc.VerifySecret(ctx, username, secret)
 	if err != nil {
+		user, err2 := uctl.GetByName(ctx, username)
```

Comment:
We should avoid calling the uctl.GetByName twice, I suggest rewriting it as following:

```
        u, err := uctl.GetByName(ctx, username)
	if err != nil {
		logger.Errorf("failed to get user model, username: %s, error: %v", username, err)
		return nil
	}

	info, err := oidc.VerifySecret(ctx, username, secret)
	if err != nil {
		if u.UserID == 1 {
			return nil
		}
		logger.Errorf("failed to verify secret, username: %s, error: %v", username, err)
		return nil
	}
```